src/DotNetWorker/Hosting/WorkerHostBuilderExtensions.cs

```diff
@@ -57,7 +57,7 @@ public static IHostBuilder ConfigureFunctionsWorkerDefaults(this IHostBuilder bu
         /// <returns>The <see cref="IHostBuilder"/>.</returns>
         public static IHostBuilder ConfigureFunctionsWorkerDefaults(this IHostBuilder builder, Action<IFunctionsWorkerApplicationBuilder> configure)
         {
-            return builder.ConfigureFunctionsWorkerDefaults(configure, o => { });
+            return builder.ConfigureFunctionsWorkerDefaults(configure, null);
```

Comment:
Just updating this for now. Anticipating the question about potentially removing the overload: this is something that was discussed when the API was first exposed and kept, so for now, just avoiding an additional registration per call.